src/components/AppNavigation/Settings/SettingsAddressbook.vue

```diff
@@ -107,14 +107,23 @@
 				</template>
 
 				<!-- delete addressbook -->
-				<ActionButton v-if="hasMultipleAddressbooks"
+				<ActionButton v-if="hasMultipleAddressbooks && addressbook.url.includes('shared_by')"
```

Comment:
let's test the principal value: https://github.com/nextcloud/calendar/blob/bb358250c2c641c967bfd66e1987c8180745b82a/src/models/calendar.js#L108C1-L114C3

Reply:
I don't have the principal for address books - mapDavCollectionToAddressbook only takes the address book. How can I add the principal?

Reply:
I don't see where it's triggered but Contacts dispatches a `PROPFIND /remote.php/dav/principals/users/admin/`. We might already have access to the data :rocket: